### PR TITLE
Make updates for AQ + cloud page for heading buttons

### PR DESCRIPTION
**Pull request by jennifer-shehane**

### CODE REVIEW

File: docs/accessibility/get-started/introduction.mdx [hunk 1]
```diff
@@ -6,22 +6,47 @@ sidebar_position: 10
...
 ---
 
-# Cypress Accessibility
+# Cypress Accessibility <Badge type="success">+ Add-on</Badge>
 
-<AccessibilityAddon />
+Instantly visualize, triage, and fix accessibility violations without any additional code or configuration.
+Dive deep into each violation with live, fully-rendered DOM snapshots of your application as it appeared during your tests.
+Track your team's progress over time with historical scores to monitor improvements and identify regressions.
+Looking for a general guide for Accessibility testing in Cypress App? [Go here.](/app/guides/accessibility-testing)
```
Comment (MikeMcC399) [inline docs/accessibility/get-started/introduction.mdx:14]:
```suggestion
Looking for a general guide for Accessibility testing in Cypress App? [Go here](/app/guides/accessibility-testing).
```

File: docs/accessibility/get-started/introduction.mdx [hunk 2]
```diff
@@ -6,22 +6,47 @@ sidebar_position: 10
...
   alt="A graphic showing the Cypress Accessibility dashboard including list of rules, views, and the user interface showing how devtools can be used"
+  noBorder={true}
 />
 
-## How are reports created?
+## Get Started
+
+You don't need to make any changes to your code or your test setup. If you record test runs to the Cypress Cloud with Test Replay, you're ready to start using Cypress Accessibility.
+You'll start your [free trial](https://www.cypress.io/accessibility?utm_medium=intro-cta&utm_source=docs.cypress.io&utm_content=Request%20trial) with personalized support
+from our sales team and your existing test data to instantly see full accessibility reports for your all Cypress projects.
```
Comment (MikeMcC399) [inline docs/accessibility/get-started/introduction.mdx:42]:
```suggestion
from our sales team and your existing test data to instantly see full accessibility reports for all your Cypress projects.
```

File: docs/accessibility/get-started/introduction.mdx [hunk 3]
```diff
@@ -6,22 +6,47 @@ sidebar_position: 10
...
+From there, you can integrate with CI to set your own standards for handling the results.
+
+## How it Works
+
+Cypress Accessibility provides organized, visual, and actionable accessibility reports, based completely on the tests you record to Cypress Cloud, and powered by Axe Core® by Deque Systems. This is the documentation page for that product. To learn about the general topic of accessibility testing using the Cypress App, see [our guide on this topic](/app/guides/accessibility-testing).
```
Comment (MikeMcC399) [inline docs/accessibility/get-started/introduction.mdx:47]: > This is the documentation page for that product. 

Confusing statement. What is the product? "Cypress Accessibility" or "Axe Core".

Perhaps this sentence could be removed?

File: docs/ui-coverage/get-started/introduction.mdx [hunk 4]
```diff
@@ -6,13 +6,37 @@ sidebar_position: 10
...
+<DocsImage
+  src="/img/ui-coverage/get-started/uicov-docs-1.gif"
+  alt="UI Coverage demo showing UI of Cloud product"
+  noBorder="true"
+/>
 
-UI Coverage is a visual test coverage report based on the interactive elements that make up your application. It highlights areas of missed test coverage directly within every page of your application, that you can use to make data-driven testing decisions. UI Coverage gives you the tools you need to understand coverage gaps, inform your testing strategy, and ship high-quality code with confidence.
+## Get Started
+
+UI Coverage works instantly, with no setup or code instrumentation required.. If you record test runs to the Cypress Cloud with Test Replay, you're ready to start using UI Coverage.
```
Comment (MikeMcC399) [inline docs/ui-coverage/get-started/introduction.mdx:37]:
```suggestion
UI Coverage works instantly, with no setup or code instrumentation required. If you record test runs to the Cypress Cloud with Test Replay, you're ready to start using UI Coverage.
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Some typo and other comments added.

File: docs/accessibility/get-started/introduction.mdx (hunk 3 above)
Comment (jennifer-shehane) [reply docs/accessibility/get-started/introduction.mdx:47]: Yah we’re going to change all of this content actually, it’s the old content I left on the bottom

**Pull request merged**